AzureSignalR.sln

```diff
@@ -73,10 +73,12 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "emulator", "emulator", "{AD
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Azure.SignalR.Emulator", "src\Microsoft.Azure.SignalR.Emulator\Microsoft.Azure.SignalR.Emulator.csproj", "{ABAAAEA8-40B3-4178-A7FD-09779A5DC757}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "ChatSample.Net50", "samples\ChatSample\ChatSample.Net50\ChatSample.Net50.csproj", "{29EF376E-DF94-419F-9DF2-50756B60DF38}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "ChatSample.Net50", "samples\ChatSample\ChatSample.Net50\ChatSample.Net50.csproj", "{29EF376E-DF94-419F-9DF2-50756B60DF38}"
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Azure.SignalR.Emulator.Tests", "test\Microsoft.Azure.SignalR.Emulator.Tests\Microsoft.Azure.SignalR.Emulator.Tests.csproj", "{870A10E3-D17A-4239-9715-5610FFE1FC76}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Azure.SignalR.Management.NetCore21.Tests", "test\Microsoft.Azure.SignalR.Management.NetCore21.Tests\Microsoft.Azure.SignalR.Management.NetCore21.Tests.csproj", "{5D557F79-88A0-429F-89C2-63BFB8260E02}"
```

Comment:
just realized that we don't need another project, right? one project can support multiple targetFrameworks

Reply:
But in the existing `Management.Test` project, there is many expressions not supported by .Net Core 2.1 and need refactoring. I think add a new project would be cleaner.

Reply:
sounds acceptable